app/auth/routes.py

```diff
@@ -59,13 +59,74 @@ def register():
             "msg": "Error in request method"
         }), 400
 
+@app.route('/login/', methods=['POST'])
+def login():
+    if current_user.is_authenticated:
+        return jsonify({
+                    'error': False,
+                    'code': 200,
+                    'title': 'User is authenticated',
+                    'msg': 'User is authenticated'
+                }),200
+    try:
+        data = request.get_json()
+    except Exception as e:
+        return jsonify({
+            'error': True,
+            'code': 400,
+            'title': 'Request failed',
+            'msg': str(e)
+            }), 400
+    username = data.get("username", None)
+    if not username:
+        return jsonify({
+            "error": True,
+            "code": 400,
+            "title": "Request failed",
+            "msg": "Username not found"
+            }), 400
+    user = User.find_by_name(username)
```

Comment:
classmethod. If you want to "search" an object by an attribute always use classmethods